contrib/cirrus/lib.sh

```diff
@@ -205,6 +208,7 @@ in_podman() {
                    -v "$HOME/auth:$HOME/auth:ro" \
                    -v /sys/fs/cgroup:/sys/fs/cgroup:rw \
                    -v /dev/fuse:/dev/fuse:rw \
+                   -v /tmp:/var/lib/containers:rw \
```

Comment:
do we even use /var/lib/containers? seems pointless if all the tests write to a tmpdir